params/config.go

```diff
@@ -793,7 +793,7 @@ func configBlockEqual(x, y *big.Int) bool {
 		return y == nil
 	}
 	if y == nil {
-		return x == nil
+		return false
```

Comment:
current code is more robust, new code depends on order. Neither is IMO more readable than the other. please revert

Reply:
done